File: w09/d03/quin_cogdell/mail_objects_app/app/assets/javascripts/mailItem.js


mailItem = function(address) {

};

mailItem.prototype.shipToAddress = function(address){
  if (typeof(address) === "string"){
    this.shipToAddress = address;
  } else {
    throw "must be a string";
  }

};

mailItem.prototype.stampValue = function(stampPrice){
  if (typeof(stampPrice) === "number"){
    this.stampValue = stampPrice;
  } else {
    throw "must be a number";
  }


};

File: w09/d01/will_smith/mail_items/lib/mailitem.rb
class Mailitem

  def initialize(options = {})
    @ship_to_address = options[:ship_to_address]
    @stamp_value = options[:stamp_value]
  end

  def ship_to_address
    @ship_to_address
  end

  def stamp_value
    @stamp_value
  end
end
